Resources/Prototypes/_RMC14/Entities/Structures/Misc/landing_signs.yml

```
@@ -1,10 +1,14 @@
 - type: entity
-  parent: [BaseStructureDynamic, CMBaseStructureCorrodible]
+  parent: CMBaseStructureCorrodible
   abstract: true
   id: RMCBaseLandingSign
   name: sign
   description: A sign.
   components:
+  - type: Transform
+    anchored: false
+  - type: Physics
+    bodyType: Dynamic
```

Actually this will still make it pullable, you want Static
```suggestion
    bodyType: Static
```